sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -3380,4 +3380,20 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
         "unsupported" -> unsupported.toString,
         "class" -> unsupported.getClass.toString))
   }
+
+  def tableFunctionDuplicateNamedArguments(name: String, pos: Int): Throwable = {
+    new AnalysisException(
```

Comment:
Thanks for the feedback! 

Honestly, I think the specific error is more like a compilation error instead of a parsing error? 

We can parse it correctly, there is no syntax error, it is more like failing to resolve/compile the given `UnresolvedTableValuedFunction` into an actual resolved function.